site/styles/_mixins.scss

```diff
@@ -23,3 +23,15 @@
     z-index: 8;
     @include gradient-x( rgba($left-gradient-bg-color, 1), transparent, 0%, 100%);
 }
+
+@mixin a-links{
+    color: $body-color;
```

Comment:
I think we should have a `$link-color` that might initially be set to `$body-color` but gives us an easy override point if we need to have it be different.